src/fund_my_watcard/logger.py

```diff
@@ -0,0 +1,49 @@
+import logging
+import colorlog
+from .config_file import CONFIG_FILE_PATH
+from .util import PRINT_PREFIX
+
+# add new level - SUCCESS
+SUCCESS = logging.ERROR - 1
+logging.addLevelName(SUCCESS, "SUCCESS")
+
+
+def success(self, msg, lvl=SUCCESS, *args, **kws):
+    self.log(lvl, msg, *args, **kws)
+
+
+logging.Logger.success = success
+
+
+def init_logger():
+    # create logger
+    logger = logging.getLogger("logger")
+    logger.setLevel(logging.INFO)
+
+    # create file handler and set level to INFO
+    logfile = CONFIG_FILE_PATH.replace("config", "log")
+    fh = logging.FileHandler(logfile)
+    fh.setLevel(SUCCESS)
```

Comment:
I change this to SUCCESS. Why we need to add all the things into log? Since we only use log to see transactions.